Fix voter card loading

ilamanov opened the pull request:
Fixes https://linear.app/agora-app/issue/AGORA-3240/troubleshoot-fix-hover-card-that-never-loads

For the default message (when the statement is `null`), I just show `No statement` as in the screen recording below. But I'm not sure if there is a better message we can display here. All the content of a valid hover card seem to be dependent on `statement` being non-null so there is really nothing we can show as a fallback.

### Demo
https://github.com/user-attachments/assets/724d1f22-6843-4b49-9cf4-38a6cf614912

jefag commented:
Glad it's not actually a real bug.  Just unfinished polish.

So there really is two cases.

# ENS Name exists, with / without a profile pic from ENS:

<img width="388" alt="image" src="https://github.com/user-attachments/assets/7153fd42-2e54-455f-a028-23c14f7f3841">

# No ENS Name exists (and therefore there would be no profile pic)

<img width="381" alt="image" src="https://github.com/user-attachments/assets/903a404d-3233-40a2-9ddb-e696d23037f8">

I think we still want the "delegate" button, because that's still technically possible. You might want to delegate to an anon, it's wierd, but its valid.

ilamanov commented:
> Glad it's not actually a real bug. Just unfinished polish.
> 
> So there really is two cases.
> 
> # ENS Name exists, with / without a profile pic from ENS:
> <img alt="image" width="388" src="https://private-user-images.githubusercontent.com/172391314/387438421-7153fd42-2e54-455f-a028-23c14f7f3841.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzIyNDUwNzgsIm5iZiI6MTczMjI0NDc3OCwicGF0aCI6Ii8xNzIzOTEzMTQvMzg3NDM4NDIxLTcxNTNmZDQyLTJlNTQtNDU1Zi1hMDI4LTIzYzE0ZjdmMzg0MS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMTIyJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTEyMlQwMzA2MThaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1jMzVkMDU2NmYwN2Y2MjQ5YjQzNTM0YTczNzlmMDk5MmM0YTNhZWUxMzQ3ZDlhMzcyZWU2MjYyMjg5ZjgzM2YxJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.EhW2uOHArZpq_jBhV4KaYlzb0vvauxtz9kKzSfCiRlw">
> # No ENS Name exists (and therefore there would be no profile pic)
> <img alt="image" width="381" src="https://private-user-images.githubusercontent.com/172391314/387438692-903a404d-3233-40a2-9ddb-e696d23037f8.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzIyNDUwNzgsIm5iZiI6MTczMjI0NDc3OCwicGF0aCI6Ii8xNzIzOTEzMTQvMzg3NDM4NjkyLTkwM2E0MDRkLTMyMzMtNDBhMi05ZGRiLWU2OTZkMjMwMzdmOC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMTIyJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTEyMlQwMzA2MThaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0xMzEwYzczMmNhY2RkOGVjODRjYTlhMjdhMmFlYTYyNmU3YWM5YjMzYmZiYmE4YmFhYzNhY2M3YThjZmQ3NGE3JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.wTstwdXggmEH3dk2cL5NSB4qps2--GSiaGWqE1MKM8Q">
> I think we still want the "delegate" button, because that's still technically possible. You might want to delegate to an anon, it's wierd, but its valid.

Thanks, fixed now!

https://github.com/user-attachments/assets/d9bb7fb6-7e85-4d58-8a22-bc4dd41f3203

I essentially kept the same rendering of the card as in the normal case (when the statement is available). Just changed the statement to "This delegate has not yet created a delegate statement."

mcgingras reviewed (approved):
This is great, I agree with the back and forth between you and Jeff. 
Thanks!
